### ISSUE TITLE
Bug in runtime enums

### ISSUE THREAD

**Issue by mariandaniel24** [RobinBlomberg/kysely-codegen#150]
Hello, in regard to the implementation of runtime enums in #115 , I believe there is a bug with the generated keys.
So, given this SQL
```sql
CREATE TYPE risk_evaluation_status AS ENUM ('OK', 'CAN_USE', 'SHOULD_NOT_BE_USED');
```
When I generate types using `kysely-codegen --camel-case --runtime-enums --dialect=postgres --out-file=generated/db-types.ts`:

The output is:
```ts
// It generates this
export enum RiskEvaluationStatus {
	CANUSE = 'CAN_USE',
	OK = 'OK',
	SHOULDNOTBEUSED = 'SHOULD_NOT_BE_USED',
}

// It should actually generate
export enum RiskEvaluationStatus {
	CAN_USE = 'CAN_USE',
	OK = 'OK',
	SHOULD_NOT_BE_USED = 'SHOULD_NOT_BE_USED',
}

```

## Upvote & Fund

- We're using [Polar.sh](https://polar.sh/RobinBlomberg) so you can upvote and help fund this issue.
- We receive the funding once the issue is completed & confirmed by you.
- Thank you in advance for helping prioritize & fund our backlog.

<a href="https://polar.sh/RobinBlomberg/kysely-codegen/issues/150">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/RobinBlomberg/kysely-codegen/issues/150/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/RobinBlomberg/kysely-codegen/issues/150/pledge.svg">
</picture>
</a>

**Comment by michael-land** [comment]
Encountering the same issue.

**Comment by acro5piano** [comment]
Same here. If it's not intended, I'll hopefully create a PR.

**Comment by RobinBlomberg** [comment]
Should be released in [kysely-codegen@0.16.0](https://github.com/RobinBlomberg/kysely-codegen/releases/tag/0.16.0)! 🚀

**Comment by acro5piano** [comment]
Thanks @RobinBlomberg
